packages/dds/tree/src/test/feature-libraries/modular-schema/modularChangeFamily.spec.ts

```diff
@@ -275,16 +283,37 @@ describe("ModularChangeFamily", () => {
 
         editor.submitChange(path, fieldB, valueField.identifier, brand(valueChange1a));
         const changes = editor.getChanges();
-        const innerChange: FieldChangeMap = new Map([[
-            fieldB,
-            { fieldKind: valueField.identifier, change: brand(valueChange1a) },
-        ]]);
+        const nodeChange: NodeChangeset = {
+            fieldChanges: new Map([[
+                fieldB,
+                { fieldKind: valueField.identifier, change: brand(valueChange1a) },
+            ]]),
+        };
 
         const expectedChange: FieldChangeMap = new Map([[
             fieldA,
-            { fieldKind: singleNodeField.identifier, change: brand(innerChange) },
+            { fieldKind: singleNodeField.identifier, change: brand(nodeChange) },
         ]]);
 
         assert.deepEqual(changes, [expectedChange]);
     });
+
+    it("build value change", () => {
+        const editor = family.buildEditor((delta) => {}, new AnchorSet());
+        const path: UpPathWithFieldKinds = {
+            parent: undefined,
+            parentField: fieldA,
+            parentFieldKind: singleNodeField.identifier,
+            parentIndex: 0,
+        };
+
+        const value = "Test Value";
+        editor.setValue(path, value);
+
+        const nodeChange: NodeChangeset = { valueChange: { value } };
+        const expectedChange: FieldChangeMap = new Map([[
+            fieldA,
+            { fieldKind: singleNodeField.identifier, change: brand(nodeChange) },
+        ]]);
```

Comment:
Is this not being used? Are we missing the assert? :)

Reply:
Thanks